src/utils/work/seekWorkResponse.js

```diff
@@ -61,11 +61,14 @@ const dispatchWorkRequest = async (
 
   const responsePromise = new Promise((resolve, reject) => {
     io.on(`WorkResponse-${ETag}`, (res) => {
-      const { response: { error } } = res;
+      const { response } = res;
+
+      if (response?.error) {
```

Comment:
Based on my knowledge, there will always be `error` in response

Reply:
Yes, "error" will always be there for work responses. I'll remove the null check (`?.`)

Reply:
Removed